packages/backend/src/execution/types.d.ts

```diff
@@ -7,10 +7,12 @@ import { QueryNode } from '../query/createQueryTree';
 /**
  * # Execution Plan
  *
- * The execution plan tree describes the steps that need to be taken to execute a query,
- * as well as which strategies are taken to execute each query.
+ * The execution plan tree describes the steps that
```

Comment:
jim what are all these formatting changes?

Reply:
Just for readability.. was having to scroll horizontally considerably, and that was hindering me grasping what I was trying to read